app/models/start_price_thresholds.rb

```diff
@@ -0,0 +1,8 @@
+class StartPriceThresholds < Struct.new(:start_price)
+  MICROPURCHASE = 3500
+  SAT = 150000
+
+  def small_business?
```

Comment:
since this is only used in one eligiblity (`small_business_eligibility.rb`) - thoughts on moving this method there?